### ISSUE TITLE
[BLUEJ-1195] Give names to text input fields

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/BlueJ-Greenfoot#1169]
For accessibility (e.g., blind users), text fields should have names. As currently implemented, the text fields do not have names so a utility like VoiceOver simply says "text field, text field, text field" as the user navigates around the UI.

JavaFX has some built-in ways of giving text fields (and other input widgets) names that VoiceOver will use when speaking to the user.

I have a blind student in my classes this term. I would be happy to contribute the changes needed if someone could point me in the right direction in the source code.

---

**Issue metadata**

- Issue type: Task
- Priority: Medium
